### PR TITLE
Document dev mode and release notes

### PR DESCRIPTION
**Pull request by alansemenov**

### CODE REVIEW

File: docs/releases.adoc [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+* Problem stopping a sandbox on Mac/Linux
+
 == v2.6.0
 
 === Features
 
 * `upgrade` command
 * `uninstall` command
 
+=== Bug
```
Comment (pmi) [inline docs/releases.adoc:24]: Remove section ?

File: docs/releases.adoc [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+* Development mode: `dev` and `project dev` commands
+
+=== Improvements
+
+* Repository and sandbox arguments for the global `create` command
+
+=== Bug fixes
+
+* Problem stopping a sandbox on Mac/Linux
+
```
Comment (pmi) [inline docs/releases.adoc:16]: Add https://github.com/enonic/cli-enonic/issues/474 ?

**Review by pmi — APPROVED** [review]

File: docs/releases.adoc (hunk 2 above)
Comment (alansemenov) [reply docs/releases.adoc:17]: Done

File: docs/releases.adoc (hunk 1 above)
Comment (alansemenov) [reply docs/releases.adoc]: Done

**Pull request merged**